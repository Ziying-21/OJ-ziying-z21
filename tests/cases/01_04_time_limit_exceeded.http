###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 385
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:48 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:18:48.766Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:49 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":124784},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:18:48.766Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:50 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":124784},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:18:48.766Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 456
# content-type: application/json
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:18:51 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":124784},{"id":1,"info":"","memory":0,"result":"Time Limit Exceeded","time":2101672}],"created_time":"2022-09-09T02:18:48.766Z","id":0,"result":"Time Limit Exceeded","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":"2022-09-09T02:18:50.997Z"}
