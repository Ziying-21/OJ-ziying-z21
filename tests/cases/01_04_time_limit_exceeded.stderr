[2022-09-09T02:18:47Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:18:47Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:18:48Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000252
[2022-09-09T02:18:48Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 385 "-" "-" 0.001647
[2022-09-09T02:18:49Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000409
[2022-09-09T02:18:50Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000236
[2022-09-09T02:18:51Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 456 "-" "-" 0.000345
